db/schema.rb

```diff
@@ -420,7 +420,7 @@
     t.string "issuer"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.index ["jti"], name: "index_security_events_on_jti"
+    t.index ["jti", "user_id", "issuer"], name: "index_security_events_on_user_id_jti_and_issuer", unique: true
```

Comment:
Do you think "event_type" should be added? Would mostly be future safety since only one event type is supported now.

Reply:
I figure no, we're just trying to guard against getting the same event twice, so it seemed safe.

Maybe I should just make it unique on `[jti, issuer]`? The goal is to prevent an SP from double-submitting, not ensure that they're globally unique

Reply:
Ah, got it. I think `['jti, 'user_id', 'issuer']` is preferable in case something caused an SP to need to send events for a bunch of users in a short timeframe?

Reply:
`jti` itself should be like a unique ID, nothing to do with time? but sure, we'll leave it as all 3 for now, and it's easier to loosen the restriction in the future than tighten it if we need to